### ISSUE TITLE
Build causes crash on 1.14.4

### ISSUE THREAD

**Issue by solidshadow96** [cabaletta/baritone#830]
TLDR: Game crashes when loading any schematic in single player world/server

Im currently using baritone through the Aristois client however I doubt this is part of the issue.

As I am new to Baritone, I have been messing around with some of the features, including its #build command, only to find that the loading of any schematic causes the game to crash, giving this error log:

[crash-2019-08-21_02.33.02-client.txt](https://github.com/cabaletta/baritone/files/3523900/crash-2019-08-21_02.33.02-client.txt)

The crash is not immediate however as the "Saving world..." screen is shown before the game closes. While this may be meaningless, I thought i'd include this in case it is of any importance, as to me, it suggests a glitch, rather than a complete break in Baritone and the game.

If theres any more information that could be of use to anyone trying to figure this out id be more than happy to provide it, as im sure im not the only person dealing with this issue :)

Thanks in advance!

**Comment by xpodo** [comment]
build only works in 1.12.2
